packages/core/CalendarManager.ts

```diff
@@ -348,6 +349,20 @@ export const updateEvent = async (
           })
       : undefined;
 
+  if (!updatedResult) {
+    logger.error(
+      "updateEvent failed",
+      safeStringify({
+        success,
+        bookingRefUid,
+        credential: getPiiFreeCredential(credential),
+        originalEvent: getPiiFreeCalendarEvent(calEvent),
+        calError,
+      })
+    );
+    throw new Error("update_calendar_event_error");
+  }
+
```

Comment:
This would throw error if it is unable to update calendar event and you would get the error message on the booking page